Tests/Set-BMVariable.Tests.ps1

```diff
@@ -285,6 +299,22 @@ Describe 'Set-BMVariable' {
         ThenNoErrorWritten
     }
 
+    It 'should convert map to OtterScript map' {

```

Comment:
Add a test for when an empty object, e.g. an empty array and hashtable, are passed to the `Set-BMVariable` function.